etc/config/cuda.amazon.properties

```diff
@@ -245,6 +245,12 @@ compiler.hipclang-rocm-50203.name=clang rocm-5.2.3
 compiler.hipclang-rocm-50203.nvdisasm=/opt/compiler-explorer/clang-rocm-50203/bin/llvm-objdump
 compiler.hipclang-rocm-50203.objdumper=/opt/compiler-explorer/clang-rocm-50203/bin/llvm-objdump
 compiler.hipclang-rocm-50203.options=-x hip --cuda-device-only -mllvm -amdgpu-early-inline-all=true -mllvm -amdgpu-function-calls=false -O3 -D__HIP_PLATFORM_AMD__=1 -D__HIP_PLATFORM_HCC__=1 -isystem /opt/compiler-explorer/libs/rocm/5.2.3/include --rocm-path=/opt/compiler-explorer/libs/rocm/5.2.3 --rocm-device-lib-path=/opt/compiler-explorer/clang-rocm-5.2.3/amdgcn/bitcode -include hip/hip_runtime.h
+compiler.hipclang-rocm-50302.exe=/opt/compiler-explorer/clang-rocm-5.3.2/bin/clang++
```

Comment:
The tests realized that `hipclang-rocm-50203` is missing from the group's `compiler=` prop list

Reply:
Sorry about that. It's a lot easier to test things once the tarballs are on S3 and I was a bit lazy on this one. Apologies. I think I fixed the issue, but I should do a bit of manual testing before this is merged.

Reply:
Ah no worries! That's what the checks are for :)